### DISCUSSION TITLE
How do I determine why my rules are listed as "potential rules" only?

### DISCUSSION THREAD

**Discussion by lee-chase** [Mergifyio/mergify#2682] [Q&A] [answered]
Not sure if I'm missing some configuration, but my rules are listed as potential rules only. How do I diagnose the reason rules are not being run?

![image](https://user-images.githubusercontent.com/15086604/119633046-436afc00-be09-11eb-8b52-63f4a68ab276.png)

**Comment by jd** [comment]
Hi @lee-chase 

Can you share your complete configuration or a link to PR?

**Comment by lee-chase** [comment, marked as answer]
Sure, I pushed the config manually having grabbed the one from our react tutorial.

 https://github.com/carbon-design-system/carbon-tutorial-vue/blob/master/.mergify.yml
 
 Test PR
 
 https://github.com/carbon-design-system/carbon-tutorial-vue/pull/1031

**Reply by sileht** [reply]
There is not CircleCI running on this PR, so none of your rules can be executed.

**Reply by lee-chase** [reply]
You're right, thanks and sorry I missed the obvious.
